src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsEvoSuiteConfigurable.kt

```diff
@@ -7,15 +7,15 @@ import javax.swing.JComponent
 /**
  * This class interacts with the other two Settings classes. It provides controller functionality for the settingsState.
  */
-class TestGenieSettingsConfigurable : Configurable {
+class SettingsEvoSuiteConfigurable : Configurable {
 
-    private var settingsComponent: TestGenieSettingsComponent? = null
+    private var settingsComponent: SettingsEvoSuiteComponent? = null
 
     /**
      * Creates a settings component that holds the panel with the settings entries, and returns this panel
      */
     override fun createComponent(): JComponent? {
```

Comment:
Could you please add @return in the KDoc?

Reply:
done

Reply:
addressed. Do you like the changes? @SergeyDatskiv @jzelenjak

Reply:
Yeah. Great job!

Reply:
Nice